config.yaml

```diff
@@ -3,6 +3,7 @@ userDataTemplates:
     package_upgrade: true
     ssh_pwauth: false
 
+serverConfigPath: "serverConfigs"
```

Comment:
nit: I think it would be good to use a path like `/path/to/serverConfigs` to emphasize the fact this field is a full path and is not relative to the binary or config file.